src/api/requests/fetchFilingSubmissionLatest.tsx

```diff
@@ -1,17 +1,128 @@
-import { request } from 'api/axiosService';
+import { getAxiosInstance, request } from 'api/axiosService';
 import type { SblAuthProperties } from 'api/useSblAuth';
+import type { AxiosResponse } from 'axios';
+import { AxiosError } from 'axios';
+import { fileSubmissionState } from 'pages/Filing/FilingApp/FileSubmission.data';
 import type { FilingPeriodType, SubmissionResponse } from 'types/filingTypes';
 import type { InstitutionDetailsApiType } from 'types/formTypes';
+import type { AxiosInstanceExtended } from 'types/requestsTypes';
+import { EightHundred, Five, One, Thirty, Two, Zero } from 'utils/constants';
+
+const MAX_RETRIES = Five;
+
+// Exponential Decay for Retry Delay
+function getRetryDelay(retry = Zero): number {
```

Comment:
I really appreciate the thought you put into this! Can we modify the `retry > One` to only start using the exponential backoff strategy until after a minute or so? Maybe every 3 seconds or so until then? A max of 24 seconds is a little long between retries, and most SBLAR files won't be done immediately. Until we have more data about the average time for validation, I'd be a little conservative with the throttling, at least till the number of beta users increases. Maybe we can make a post-mvp ticket to look at the the data here and adjust the logic accordingly?

Nit: I really like the thought that went into this! I don't know if we need to use an exponential backoff strategy until at least we exceed the average wait time though? I think we want something a little different than normal timeout logic since we're dealing with something that takes some time. I don't think we need to think about this too deeply until post-mvp, but if the average wait time for a file to be validated was `X`, then isn't that around the time when we should want the most retries for the best user experience with the least amount of unnecessary hitting of the backend? 


<img width="1012" alt="shapes at 24-04-08 17 27 10" src="https://github.com/cfpb/sbl-frontend/assets/19983248/02c6850c-ccf2-4177-b3d2-e4822a1bb3ff">


I could even imagine that "average validation time" being specific to how large the file they choose to upload is, and this all assume a normal distribution. Which is all to say... let's not throttle the retries too quickly right now, until we know about how long the validation step happens for more "real" data. Also, let's try not to overthink it yet until the data comes in. 😅

Reply:
TL;DR: Maybe extend the time until the timeout algo kicks in. I'm sorry for nerdin' it up lol! 🤓📈